app/src/main/res/raw/sample2.csv

```diff
@@ -1,407 +0,0 @@
-2490,2019-08-16 09:36:04,4235.897436,22.67206349,22.67206349,,,,,,
-2491,2019-08-16 09:36:04,4226.153846,22.67206349,22.67206349,,,,,,
-2491,2019-08-16 09:36:04,4215.897436,22.67206349,22.67206349,,,,,,
-2492,2019-08-16 09:36:04,4217.948718,22.67206349,22.67206349,,,,,,
-2492,2019-08-16 09:36:04,4224.102564,22.67206349,22.67206349,,,,,,
-2493,2019-08-16 09:36:04,4217.435897,22.67206349,22.67206349,,,,,,
-2493,2019-08-16 09:36:04,4212.820513,22.67206349,22.67206349,,,,,,
-2494,2019-08-16 09:36:04,4210.769231,22.67206349,22.67206349,,,,,,
-2494,2019-08-16 09:36:04,4204.102564,22.67206349,22.67206349,,,,,,
-2494,2019-08-16 09:36:04,4196.923077,22.67206349,22.67206349,,,,,,
-2495,2019-08-16 09:36:04,4193.846154,22.67206349,22.67206349,,,,,,
-2495,2019-08-16 09:36:04,4201.025641,22.67206349,22.67206349,,,,,,
-2496,2019-08-16 09:36:04,4205.641026,22.67206349,22.67206349,,,,,,
-2496,2019-08-16 09:36:04,4205.641026,22.67206349,22.67206349,,,,,,
-2497,2019-08-16 09:36:04,4210.769231,22.67206349,22.67206349,,,,,,
-2497,2019-08-16 09:36:04,4207.692308,22.67206349,22.67206349,,,,,,
-2498,2019-08-16 09:36:04,4196.923077,22.67206349,22.67206349,,,,,,
-2498,2019-08-16 09:36:04,4187.692308,22.67206349,22.67206349,,,,,,
-2499,2019-08-16 09:36:04,4189.74359,22.67206349,22.67206349,,,,,,
-2499,2019-08-16 09:36:04,4201.025641,22.67206349,22.67206349,,,,,,
-2500,2019-08-16 09:36:04,4203.589744,22.67206349,22.67206349,,,,,,
-2500,2019-08-16 09:36:04,4202.051282,22.67206349,22.67206349,,,,,,
-2501,2019-08-16 09:36:04,4203.076923,22.67206349,22.67206349,,,,,,
-2501,2019-08-16 09:36:04,4200,22.67206349,22.67206349,,,,,,
-2502,2019-08-16 09:36:04,4198.974359,22.67206349,22.67206349,,,,,,
-2502,2019-08-16 09:36:04,4195.897436,22.67206349,22.67206349,,,,,,
-2503,2019-08-16 09:36:04,4188.717949,22.67206349,22.67206349,,,,,,
-2503,2019-08-16 09:36:04,4187.179487,22.67206349,22.67206349,,,,,,
-2504,2019-08-16 09:36:04,4185.128205,22.67206349,22.67206349,,,,,,
-2504,2019-08-16 09:36:04,4180,22.67206349,22.67206349,,,,,,
-2504,2019-08-16 09:36:04,4180.512821,22.67206349,22.67206349,,,,,,
-2505,2019-08-16 09:36:04,4192.820513,22.67206349,22.67206349,,,,,,
-2505,2019-08-16 09:36:04,4197.435897,22.67206349,22.67206349,,,,,,
-2506,2019-08-16 09:36:04,4186.153846,22.67206349,22.67206349,,,,,,
-2506,2019-08-16 09:36:04,4178.974359,22.67206349,22.67206349,,,,,,
-2507,2019-08-16 09:36:04,4166.666667,22.67206349,22.67206349,,,,,,
-2507,2019-08-16 09:36:04,4155.897436,22.67206349,22.67206349,,,,,,
-2508,2019-08-16 09:36:04,4162.564103,22.67206349,22.67206349,,,,,,
-2508,2019-08-16 09:36:04,4165.128205,22.67206349,22.67206349,,,,,,
-2509,2019-08-16 09:36:04,4167.692308,22.67206349,22.67206349,,,,,,
-2509,2019-08-16 09:36:04,4180,22.67206349,22.67206349,,,,,,
-2510,2019-08-16 09:36:04,4185.641026,22.67206349,22.67206349,,,,,,
-2510,2019-08-16 09:36:04,4181.538462,22.67206349,22.67206349,,,,,,
-2511,2019-08-16 09:36:04,4183,,,,,,,,
```

Comment:
These sample csv files are not redundant. These are used once app is installed and users get to work with these sample data. Don't delete them.